packages/shorebird_code_push_client/lib/src/code_push_client.dart

```diff
@@ -46,6 +46,35 @@ class CodePushClient {
 
   Map<String, String> get _apiKeyHeader => {'x-api-key': _apiKey};
 
+  /// Create a new artifact for a specific [patchId].
+  Future<Artifact> createArtifact({
+    required String artifactPath,
+    required int patchId,
+    required String arch,
+    required String platform,
+    required String hash,
+  }) async {
+    final request = http.MultipartRequest(
+      'POST',
+      Uri.parse('$hostedUri/api/v1/artifacts'),
+    );
+    final file = await http.MultipartFile.fromPath('file', artifactPath);
+    request.files.add(file);
+    request.fields.addAll({
+      'patch_id': '$patchId',
```

Comment:
patchid is different from patch_number correct?

Reply:
Yup, patch_number is not unique and is relative to the release